websites/F/Fandom/dist/metadata.json

```diff
@@ -28,5 +28,19 @@
 		"tv",
 		"movies",
 		"anime"
+	],
+	"settings": [
+		{
+			"id": "buttons",
+			"title": "Show buttons",
```

Comment:
```suggestion
			"title": "Show Buttons",
```

Reply:
I saw the PreMiD presence use sentence case and it is a mix on the YouTube presence. 

Are you sure about this? Please confirm again.

Reply:
Edited. By the way, would you like "Show Buttons" or "Show Button", since there is only one on every page. This is also for "Show Timestamps"

Reply:
show button wouldn't be correct english